template/.github/workflows/shipwright-test.yml

```diff
@@ -0,0 +1,23 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Shipwright Tokens Example
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  pull_request:
+    branches: ["main"]
```

Comment:
I haven't looked into the methods available in this file. If you use `pull_request` does that mean the action runs on both push and pull? The comment makes it seem like that would be the case

Reply:
Oh the comment is leftover, I removed the "push" key

it used to look like 
```yaml
push:
  branches: ["main"]
pull_request:
  branches: ["main"]